readme.md

```diff
@@ -139,7 +176,15 @@ Define a Javascript constructor called 'Instructor'. Every instance of Instructo
 Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin Laying Deer Figurine, Gold" as the argument.
 
 ```js
-// Your code starts here...
+// var instructor {
+  name: "",
+  receivePresent: function(gift){
+    console.log(name + " promptly drops " + .this + " on the floor")
+  }
+
+Andy.receivePresent("Resin Laying Deer Figurine, Gold")
+
+}
 
 ```
 
```

Comment:
A constructor is a function, not an object. So when defining any property or method inside of a construct, we use the following notation...

```js
var Instructor = function( name ){
  this.name = name;
}
```

Then, in order for that `Andy` variable to represent an instructor, you need to generate an instructor using the `new` keyword like so...

```js
var andy = new Instructor( "Andy" );
```

On a separate note, while we are allowed to define `receivePresent` inside of the constructor, it would be better to define it as a prototypal method. The primary reasons here are reducing memory usage and organization.

```js
Instructor.prototype.receivePresent = function(gift){
  console.log(this.name + " promptly drops the " + gift + " on the floor.");
}
```